frontend/src/Editor/WidgetManager/widgetConfig.js

```diff
@@ -482,7 +482,7 @@ export const widgets = [
       },
       events: [],
       styles: {
-        backgroundColor: { value: '' },
+        backgroundColor: { value: '#fff' },
```

Comment:
The issue was for setting the **default text color** not the background color.
```suggestion
        textColor: { value: '#000' },
```